README.md

```diff
@@ -60,6 +60,10 @@ $ solidus_starter_frontend
 
 The last command will copy all the necessary files.
 
+**IMPORTANT**: if you plan to use `solidus_auth_devise` you also need to patch
+`SolidusSupport.frontend_available?` so it returns `true`, otherwise authentication
```

Comment:
This is already implemented [here](https://github.com/nebulab/solidus_starter_frontend/blob/master/lib/solidus_starter_frontend/solidus_support_extensions.rb#L6).

Reply:
Did you still get issues with auth routes?

Reply:
Nevermind. I saw your issue #137
 
Can you link it in PR description?